lib/web/sprite.js

```diff
@@ -70,7 +70,7 @@ function baseUrlWorkAround(svg, currentUrlPrefix, newUrlPrefix) {
       var attributeName = attribute.localName.toLowerCase();
 
       if (fixAttributes.indexOf(attributeName) !== -1) {
-        var match = URI_FUNC_REGEX.exec(node.getAttribute(attributeName));
+        var match = URI_FUNC_REGEX.exec(node.getAttribute(attributeName)).replace(/\\(.)/mg, "$1");
```

Comment:
1. Is there other way to get unescaped attribute value?
2. Could just we turn `\\` into empty string instead?
3. Is `m` flag required?